<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::SystemConfigurationWidget</class>
 <widget class="QDialog" name="snowgui::SystemConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="snowgui::serverconfigurationwidget" name="serverTab">
      <attribute name="title">
       <string>Server</string>
      </attribute>
     </widget>
     <widget class="snowgui::remoteconfigurationpanel" name="remoteTab">
      <attribute name="title">
       <string>Remote</string>
      </attribute>
     </widget>
     <widget class="snowgui::localconfigurationwidget" name="localTab">
      <attribute name="title">
       <string>Local</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::serverconfigurationwidget</class>
   <extends>QWidget</extends>
   <header location="global">serverconfigurationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::localconfigurationwidget</class>
   <extends>QWidget</extends>
   <header location="global">localconfigurationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::remoteconfigurationpanel</class>
   <extends>QWidget</extends>
   <header location="global">remoteconfigurationpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
